numpy/core/src/npymath/npy_math.c.src

```diff
@@ -442,6 +443,26 @@ double npy_log2(double x)
     return x*DEG2RAD;
 }
 
+@type@ npy_wrap_to_pi@c@(@type@ x)
+{
+    if (x < -NPY_PI || x > NPY_PI)
+        return npy_wrap_to_2pi(x + NPY_PI) - NPY_PI;
```

Comment:
`return remainder(x, NPY_PI)` is enough here, i think - although that [needs C99](http://en.cppreference.com/w/c/numeric/math/remainder)